src/main/java/pl/widulinski/scrapp/createExcel/CreateExcel.java

```diff
@@ -0,0 +1,25 @@
+package pl.widulinski.scrapp.createExcel;
+
+
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.springframework.stereotype.Component;
+import pl.widulinski.scrapp.FoundWebElements.FoundWebElement;
+import java.io.IOException;
+import java.util.stream.Stream;
+
+@Component
+public class CreateExcel extends CreateWorkbook {
```

Comment:
Widzę że w całym pakiecie `createExcel` były jakieś próby uwspólnienia logiki. Nie wiem natomiast co chciałbyś uwspólniać. Abstrakcje i interfejsy tworzymy głownie po to gdy mamy kilka różnych implementacji które podmieniamy w locie w zależności od parametrów. Potem korzystamy z tych abstrakcji a nie konkretnych klas. 
Tutaj masz dokładnie jedną scieżke którą może pójść aplikacja - wołanie `printExcel`. Nic sie nie zmienia w zależnosci od parametrów. Więc wszystkie metody z abstrakcji przeniósł bym do tej klasy. 
Innymi słowy ta klasa do zostawienia - reszta do wyrzucenia

Reply:
zrobione